assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -0,0 +1,186 @@
+ */
+import { useCallback } from 'react';
+import { TypographyPresets } from '../../../components';
+import { Close as CloseSVG } from '../../../icons';
+import { ICON_METRICS } from '../../../constants';
+import useTelemetryOptIn from './useTelemetryOptIn';
+
+const Banner = styled.div.attrs()`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 161px;
+  margin: 0 20px;
+  background-image: url('/wp-content/plugins/web-stories/assets/images/analytics-banner-bg.png');
+  border-radius: 8px;
+`;
+
+const Header = styled.div`
+  width: 100%;
+  text-align: center;
+`;
+
+const Title = styled.h1`
+  display: inline-block;
+  font-size: 18px;
+  font-weight: ${({ theme }) => theme.typography.weight.normal};
+  line-height: 24px;
+  margin-bottom: 13px;
+`;
+
+const Label = styled.label.attrs({ htmlFor: 'telemetry-banner-opt-in' })`
+  display: flex;
+`;
+
+export const LabelText = styled.span`
+  ${TypographyPresets.Small};
+  color: ${({ theme }) => theme.colors.gray400};
+  margin-bottom: 16px;
+  max-width: 530px;
+`;
+
+const VisitSettingsText = styled(LabelText)``;
+
+const CheckBox = styled.input.attrs({
+  type: 'checkbox',
+  id: 'telemetry-banner-opt-in',
+})`
+  height: 18px;
+  width: 18px;
+  margin: 5px 12px 0 0;
+`;
+
+const CloseIcon = styled(CloseSVG).attrs(ICON_METRICS.TELEMETRY_BANNER_EXIT)`
+  color: ${({ theme }) => theme.colors.white};
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+const ToggleButton = styled.button.attrs({
+  ['aria-label']: __('Close Telemetry Banner', 'web-stories'),
+})`
+  border: none;
+  padding: 4px;
+  border-radius: 50%;
+  background: ${({ theme }) => theme.colors.gray200};
+  cursor: pointer;
+  float: right;
+  margin-right: 14px;
+
+  &:hover svg {
+    color: ${({ theme }) => theme.colors.gray50};
+  }
+  &:active svg {
+    color: ${({ theme }) => theme.colors.gray50};
+  }
+`;
+
+export function TelemetryOptInBanner({
+  visible = true,
+  disabled = false,
+  onChange = () => {},
+  onClose = () => {},
+  checked = false,
+}) {
+  return visible ? (
+    <Banner>
+      <Header>
+        <Title>{__('Help improve the editor!', 'web-stories')}</Title>
+        <ToggleButton onClick={onClose}>
+          <CloseIcon />
+        </ToggleButton>
+      </Header>
+      <Label>
+        <CheckBox checked={checked} disabled={disabled} onChange={onChange} />
+        <LabelText aria-checked={checked}>
+          {__(
+            'Check the box to help us improve the Web Stories plugin by allowing tracking of product usage stats. All data are treated in accordance with',
+            'web-stories'
+          )}
+          &nbsp;
+          <a
+            href={__('https://policies.google.com/privacy', 'web-stories')}
+            rel="noreferrer"
+            target="_blank"
+          >
+            {__('Google Privacy Policy', 'web-stories')}
+            {'.'}
+          </a>
+        </LabelText>
```

Comment:
Currently we can't i18n split sentences. I did this too on my settings checkbox PR. Pascal recommended breaking it up into a complete sentence, plus a learn more link.

https://github.com/google/web-stories-wp/pull/4330/files#diff-ce22f956eb89ee0ebb00e3ab68e7d160R70-R84

Reply:
https://github.com/google/web-stories-wp/pull/4330#discussion_r483030051

Reply:
Updated 👍